miqa/core/models/scan_note.py

```diff
@@ -15,6 +15,7 @@ class Meta:
     # We want to set created to arbitrary values during import
     created = models.DateTimeField(default=timezone.now)
     modified = ModificationDateTimeField()
-    creator = models.ForeignKey(User, on_delete=models.PROTECT)
+    creator = models.ForeignKey(User, on_delete=models.PROTECT, null=True, blank=True)
+    initials = models.CharField(max_length=2)
```

Comment:
It looks like the default value listed in the migration is different from the default value listed here. I don't really get how the checkmigrations step is passing... what gives?

Reply:
The migration default is only necessary because it's adding a non-null field with no default. When the migration is applied something needs to be put into the new column for existing rows, so I just decided it would be `??`.

Reply:
Oh I see, that makes sense, thanks.